include/libopencm3/stm32/fsmc.h

```diff
@@ -23,16 +23,18 @@
 #include <libopencm3/cm3/common.h>
 #include <libopencm3/stm32/memorymap.h>
 
-#if defined(STM32F4)
+#if   defined(STM32F4)
 #       include <libopencm3/stm32/f4/fmc.h>
+#elif defined(STM32F7)
+#       include <libopencm3/stm32/f7/fmc.h>
 #endif
 
 /* --- Convenience macros -------------------------------------------------- */
-
-#define FSMC_BANK1_BASE			0x60000000U /* NOR / PSRAM */
-#define FSMC_BANK2_BASE			0x70000000U /* NAND flash */
-#define FSMC_BANK3_BASE			0x80000000U /* NAND flash */
-#define FSMC_BANK4_BASE			0x90000000U /* PC card */
+// Those macros are not constant along mcus!
+//#define FSMC_BANK1_BASE			0x60000000U /* NOR / PSRAM */
+//#define FSMC_BANK2_BASE			0x70000000U /* NAND flash */
+//#define FSMC_BANK3_BASE			0x80000000U /* NAND flash */
+//#define FSMC_BANK4_BASE			0x90000000U /* PC card */
```

Comment:
we don't commit commented out code like this please.

Reply:
Also, you've commented them out, but I dont' see where you've put them back again for f4 and f7?

Reply:
the f4 and f7 headers are now combined